django-stubs/db/models/constraints.pyi

```diff
@@ -18,7 +18,7 @@ class BaseConstraint:
     violation_error_code: str | None
     violation_error_message: _StrOrPromise | None
     default_violation_error_message: _StrOrPromise
-    def __init__(self, name: str, violation_error_message: _StrOrPromise | None = ...) -> None: ...
+    def __init__(self, *, name: str, violation_error_message: _StrOrPromise | None = ...) -> None: ...
```

Comment:
I think it would be correct to use `@deprecated` here inside an `@overload` with `name: str, *args: Any`